src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -514,14 +534,30 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.MP4;
 		} else if (value.contains("mjpg") || value.contains("m-jpeg")) {
 			format = FormatConfiguration.MJPEG;
-		} else if (value.contains("div") || value.contains("dx")) {
-			format = FormatConfiguration.DIVX;
+		} else if (
+				value.startsWith("div") ||
+				value.equals("dx50") ||
+				value.equals("dvx1")
+			) {
+				format = FormatConfiguration.DIVX;
+		} else if (value.startsWith("indeo")) { // Intel Indeo Video: IV31, IV32, IV41 and IV50
+			format = FormatConfiguration.INDEO;
+		} else if (value.equals("yuv")) {
+			format = FormatConfiguration.YUV;
+		} else if (value.equals("rgb") || value.equals("rgba")) {
```

Comment:
As you already know, i don't appreciate that you don't even bother looking at the links that i take the effort to post :disappointed: 
That's said, i should remember that you're near always doing that...
[VRMLuncompressed.avi.txt](https://github.com/UniversalMediaServer/UniversalMediaServer/files/1143491/VRMLuncompressed.avi.txt)
[miss_am.avi.txt](https://github.com/UniversalMediaServer/UniversalMediaServer/files/1143492/miss_am.avi.txt)